src/simulation/m_riemann_solvers.fpp

```diff
@@ -544,6 +543,7 @@ contains
                                     G_R = G_R + alpha_R(i)*Gs(i)
                                 end do
 
+                                !$acc loop seq
                                 do i = 1, strxe - strxb + 1
                                     tau_e_L(i) = qL_prim_rs${XYZ}$_vf(j, k, l, strxb - 1 + i)
                                     tau_e_R(i) = qR_prim_rs${XYZ}$_vf(j + 1, k, l, strxb - 1 + i)
```

Comment:
the lines below this (560-580 or so are all commented out... why? remove or keep...)

Reply:
I updated this part of the code for RMT with HLL, this has not been tested. 

I will build test cases for this and HLLC with RMT over the next few days.